Upgrade.md

```diff
@@ -6,6 +6,39 @@ break. For the full list of changes, please look at the Changelog file.
 
 ## 2.0.0-alpha1 to 2.0.0-alpha4
```

Comment:
should be alpha3 to alpha4 (or we should have more sections if some BC breaks are older than alpha3)